tests/testthat/test-epinowcast.R

```diff
@@ -11,6 +11,13 @@ if (not_on_cran() & on_ci()) {
   }
 }
 
+# A function to filter data used in test suite
+run_window_filter <- function(x, filter_report_remove = 10, filter_reference_include = 10) {
```

Comment:
TBH this is quite a nice helper function - maybe its something to think about for the package.

Reply:
Happy to move to an issue?

Reply:
See #404